SignalMessaging/ViewControllers/ApprovalFooterView.swift

```diff
@@ -127,19 +127,36 @@ public class ApprovalFooterView: UIView {
     }()
 
     lazy var proceedButton: OWSButton = {
-        let button = OWSButton.sendButton(imageName: proceedImageName) { [weak self] in
+		let button = OWSButton.sendButton(
+			imageName: self.approvalMode.proceedButtonImageName
+		) { [weak self] in
             guard let self = self else { return }
             self.delegate?.approvalFooterDelegateDidRequestProceed(self)
         }
 
         return button
     }()
 
-    private var proceedImageName: String {
-        return approvalMode == .send ? "send-solid-24" : "arrow-right-24"
```

Comment:
Nit: using a ternary operator here was functionally fine, but if/when a third case is ever added to the `ApprovalMode` enum, this ternary wouldn't catch the problem.

By moving this to a var on the enum with a `switch` statement, we'll get a nice compiler error if/when the enum expands in the future.

(This is optional! Just a little tidying while I was in here.)